### PR TITLE
🛢 Auto-publish installers at publish datetime

### PR DESCRIPTION
**Pull request by bethbrains**

### CODE REVIEW

File: metadeploy/api/jobs.py [hunk 1]
```diff
@@ -296,6 +296,16 @@ def enqueuer():
 enqueuer_job = job(enqueuer)
 
 
+def publish_installers():
+    now = timezone.now()
+    for v in Version.objects.filter(publish_date__lt=now, is_listed=False):
```
Comment (davisagli) [inline metadeploy/api/jobs.py:301]: Does this include publish_date of None or not? We don't want to.

File: metadeploy/api/serializers.py [hunk 2]
```diff
@@ -271,6 +271,7 @@ class Meta:
             "primary_plan",
             "secondary_plan",
             "is_listed",
+            "publish_date",
```
Comment (davisagli) [inline metadeploy/api/serializers.py:274]: @davidmreed Does the frontend need access to publish_date? Admin access is via a separate API in https://github.com/SFDO-Tooling/MetaDeploy/blob/main/metadeploy/adminapi/api.py and should automatically include the new field.

**Comment by davisagli** [comment]
(Added do-not-merge label to remind us this is pending security assessment)

File: metadeploy/api/jobs.py (hunk 1 above)
Comment (bethbrains) [reply metadeploy/api/jobs.py:301]: Added a test case to cover this.

File: metadeploy/api/serializers.py (hunk 2 above)
Comment (davidmreed) [reply metadeploy/api/serializers.py:274]: No, it doesn't. Thanks for clarifying that! I did not realize the admin API was separate.

**Review by davidmreed — APPROVED** [review]

**Pull request merged**